com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl

```diff
@@ -134,18 +132,14 @@ bool TryToGetPoolUVWAndSubdiv(APVResources apvRes, float3 posWS, float3 normalWS
     bool hasValidUVW = true;
 
     // transform into APV space
-    float3 posRS = mul(_WStoRS, float4(posWS + normalWS * _NormalBias
-                                             + viewDirWS * _ViewBias, 1.0)).xyz;
+    float3 posRS = mul(_WStoRS, float4(posWS + normalWS * _NormalBias + viewDirWS * _ViewBias, 1.0)).xyz;
```

Comment:
Why changing this? It is a long line and to me in a single line makes it a bit harder to parse